marketing_transactional.go

```diff
@@ -0,0 +1,50 @@
+package hubspot
+
+import "fmt"
+
+const transactionalBasePath = "transactional"
+
+// TransactionalService is an interface for the marketing/transactional service of the HubSpot API.
+// Reference: https://developers.hubspot.com/docs/api/marketing/transactional-emails
+type TransactionalService interface {
+	SendSingleEmail(props *SendSingleEmailProperties) (*SendSingleEmailResponse, error)
+}
+
+// Provides the default implementation of TransactionService.
```

Comment:
It is recommended that godoc begin with a struct name. (or function name, interface name etc..)
```suggestion
// TransactionalServiceOp provides the default implementation of TransactionService.
```